src/content/docs/2/guide/frontend/trunk.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Trunk
+i18nReady: true
+---
+
+Trunk is a WASM web application bundler for Rust. Learn more about Trunk at https://trunkrs.dev. This guide is accurate as of Trunk 0.17.5.
+
+:::caution[Note on Mobile Support]
+
+Tauri mobile development is not currently possible with the official version of Trunk. If developing for mobile you will need to use https://github.com/amrbashir/trunk until https://github.com/thedodd/trunk/pull/494 and https://github.com/thedodd/trunk/pull/500 are merged and released.
+
+You will need to use `cargo install --git https://github.com/amrbashir/trunk` when installing Trunk and then set `serve.ws_protocol = "ws"` in `Trunk.toml`.
+
+:::
+
+## Checklist
+
+- Use SSG, Tauri doesn't officially support server based solutions.
+- Use `address = "0.0.0.0"` so that the webserver is available on the network for mobile development.
```

Comment:
```suggestion
- Use `address = "0.0.0.0"` so that the webserver is available on the network for mobile development.
- Enable `withGlobalTauri` to ensure the Tauri API's are available in the `window.__TAURI__` variable so you can import them using `wasm-bindgen`.
```